assets/queries/terraform/aws/cmk_rotation_disabled/query.rego

```diff
@@ -17,21 +19,48 @@ CxPolicy[result] {
 	}
 }
 
+
+CxPolicy[result] {
+	resource := input.document[i].resource.aws_kms_key[name]
+
+	not key_set_to_false(resource)
+    resource.enable_key_rotation == true
+    common_lib.valid_key(resource, "customer_master_key_spec")
```

Comment:
@LupovichRan, sorry for requesting another change. I forgot to ask you to delete this line

```suggestion
```